contracts/SSVNetwork.sol

```diff
@@ -327,31 +532,113 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
 
         Pod memory pod = _pods[keccak256(abi.encodePacked(msg.sender, fromClusterId))];
         uint64 podIndex = _clusterCurrentIndex(fromClusterId);
-        pod.usage.balance = _ownerPodBalance(pod, podIndex) + amount;
+        pod.usage.balance = _ownerPodBalance(pod, podIndex);
         pod.usage.index = podIndex;
         pod.usage.block = uint64(block.number);
         pod.validatorCount -= activeValidatorCount;
 
         pod = _pods[keccak256(abi.encodePacked(msg.sender, toClusterId))];
         podIndex = _clusterCurrentIndex(toClusterId);
-        pod.usage.balance = _ownerPodBalance(pod, podIndex) + amount;
+        pod.usage.balance = _ownerPodBalance(pod, podIndex);
         pod.usage.index = podIndex;
         pod.usage.block = uint64(block.number);
         pod.validatorCount += activeValidatorCount;
 
-        if (_liquidatable(pod.usage.balance, pod.validatorCount, newOperatorIds)) {
-            revert AccountLiquidatable();
+        if (_liquidatable(pod.disabled, _ownerPodBalance(pod, podIndex), pod.validatorCount, newOperatorIds)) {
+            revert PodLiquidatable();
         }
     }
 
-
     // TODO add external functions below to interface
 
+    // @dev external dao functions
+
+    function getOperatorFeeIncreaseLimit() external view returns (uint64) {
+        return _operatorMaxFeeIncrease;
+    }
+
+    function getExecuteOperatorFeePeriod() external view returns (uint64) {
+        return _executeOperatorFeePeriod;
+    }
+
+    function getDeclaredOperatorFeePeriod() external view returns (uint64) {
+        return _declareOperatorFeePeriod;
+    }
+
+    function getNetworkFee() external view returns (uint256) {
+        return _networkFee.expand();
+    }
+
+    function getNetworkBalance() external onlyOwner view returns (uint256) {
+        DAO memory dao = _dao;
+        return _networkBalance(dao).expand();
+    }
+
+    function updateOperatorFeeIncreaseLimit(uint64 newOperatorMaxFeeIncrease) external onlyOwner {
+        _updateOperatorFeeIncreaseLimit(newOperatorMaxFeeIncrease);
+    }
+
+    function updateDeclareOperatorFeePeriod(uint64 newDeclareOperatorFeePeriod) external onlyOwner {
+        _updateDeclareOperatorFeePeriod(newDeclareOperatorFeePeriod);
+    }
+
+    function updateExecuteOperatorFeePeriod(uint64 newExecuteOperatorFeePeriod) external onlyOwner {
+        _updateExecuteOperatorFeePeriod(newExecuteOperatorFeePeriod);
+    }
+
+    function updateNetworkFee(uint256 fee) external onlyOwner {
```

Comment:
@andrew-blox lets add a task to have dao role and owner role

Reply:
added:
https://bloxxx.atlassian.net/browse/IO1-1979